pkg/fakerp/clusterstatus.go

```diff
@@ -0,0 +1,47 @@
+package fakerp
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/plugin"
+)
+
+// handleClusterStatus handles admin requests for the status of control plane pods
+func (s *Server) handleClusterStatus(w http.ResponseWriter, req *http.Request) {
+	cs := s.read()
+	if cs == nil {
+		s.internalError(w, "Failed to read the internal config")
+		return
+	}
+
+	config, err := GetPluginConfig()
```

Comment:
more copy/paste.  Please get to https://github.com/openshift/openshift-azure/issues/1049.